### ISSUE TITLE
Ignores Papirus icon theme

### ISSUE THREAD

**Issue by teresaejunior** [syncthing/syncthing-gtk#386]
I have currently the icon theme Papirus installed system-wide, and I did a bit of searching, and found that syncthing-gtk should be using them. The settings are all the default!

```
$ find /usr/share/icons/Papirus/ -name 'si-*' | sort -V
/usr/share/icons/Papirus/22x22/panel/si-idle.svg
/usr/share/icons/Papirus/22x22/panel/si-syncing-0.svg
/usr/share/icons/Papirus/22x22/panel/si-syncing-1.svg
/usr/share/icons/Papirus/22x22/panel/si-syncing-2.svg
/usr/share/icons/Papirus/22x22/panel/si-syncing-3.svg
/usr/share/icons/Papirus/22x22/panel/si-syncing-4.svg
/usr/share/icons/Papirus/22x22/panel/si-syncing-5.svg
/usr/share/icons/Papirus/22x22/panel/si-syncing-6.svg
/usr/share/icons/Papirus/22x22/panel/si-syncing-7.svg
/usr/share/icons/Papirus/22x22/panel/si-syncing-8.svg
/usr/share/icons/Papirus/22x22/panel/si-syncing-9.svg
/usr/share/icons/Papirus/22x22/panel/si-syncing-10.svg
/usr/share/icons/Papirus/22x22/panel/si-syncing-11.svg
/usr/share/icons/Papirus/22x22/panel/si-unknown.svg
/usr/share/icons/Papirus/24x24/panel/si-idle.svg
/usr/share/icons/Papirus/24x24/panel/si-syncing-0.svg
/usr/share/icons/Papirus/24x24/panel/si-syncing-1.svg
/usr/share/icons/Papirus/24x24/panel/si-syncing-2.svg
/usr/share/icons/Papirus/24x24/panel/si-syncing-3.svg
/usr/share/icons/Papirus/24x24/panel/si-syncing-4.svg
/usr/share/icons/Papirus/24x24/panel/si-syncing-5.svg
/usr/share/icons/Papirus/24x24/panel/si-syncing-6.svg
/usr/share/icons/Papirus/24x24/panel/si-syncing-7.svg
/usr/share/icons/Papirus/24x24/panel/si-syncing-8.svg
/usr/share/icons/Papirus/24x24/panel/si-syncing-9.svg
/usr/share/icons/Papirus/24x24/panel/si-syncing-10.svg
/usr/share/icons/Papirus/24x24/panel/si-syncing-11.svg
/usr/share/icons/Papirus/24x24/panel/si-unknown.svg
```

**Comment by kozec** [comment]
> /usr/share/icons/Papirus/24x24/panel/si-syncing-7.svg

Correct path should be `$THEME//24x24/status/si-test-unknown.png`, I'm not sure if icon will be picked in panel directory.

**Comment by teresaejunior** [comment]
I have just run `mv si-* ../status/` in both 22 and 24 directories, and then run `gtk-update-icon-cache $PWD` in the Papirus directory. After that, I removed ~/.config/syncthing-gtk/ to go back to the defaults. Still, the icon theme is not recognized! Does syncthing-gtk expect the icons to be PNG too?

**Comment by kozec** [comment]
Sorry for late answer, I had to test it. SVGs are fine and I found it's OK even to leave them in that `panel` directory.

In fact, after installing papirus-icon-theme-git package on my Manjaro XFCE machine, Syncthing icons changed exactly as they were supposed to. Only only problem I can imagine is ST-GTK not refreshing icon because there is no status change triggering it. In that case, restarting ST-GTK should be enough.
